esper/controllers/enterprise/enterprise.py

```diff
@@ -29,42 +29,40 @@ def _enterprise_basic_response(self, enterprise, format=OutputFormat.TABULATED):
         email = None
         contact_person = None
         contact_number = None
-        if enterprise.details:
-            registered_name = enterprise.details.registered_name
-            address = enterprise.details.registered_address
-            location = enterprise.details.location
-            zipcode = enterprise.details.zipcode
-            email = enterprise.details.contact_email
-            contact_person = enterprise.details.contact_person
-            contact_number = enterprise.details.contact_number
+        if enterprise:
+            registered_name = enterprise.registered_name
+            address = enterprise.registered_address
+            location = enterprise.location
+            zipcode = enterprise.zipcode
+            email = enterprise.contact_email
+            contact_person = enterprise.contact_person
+            contact_number = enterprise.contact_number
 
         if format == OutputFormat.TABULATED:
             title = "TITLE"
             details = "DETAILS"
             renderable = [
-                {title: 'id', details: enterprise.id},
-                {title: 'name', details: enterprise.name},
-                {title: 'display_name', details: enterprise.display_name},
-                {title: 'registered_name', details: registered_name},
-                {title: 'address', details: address},
-                {title: 'location', details: location},
-                {title: 'zipcode', details: zipcode},
-                {title: 'email', details: email},
-                {title: 'contact_person', details: contact_person},
-                {title: 'contact_number', details: contact_number}
+                {title: 'Enterprise Id', details: enterprise.id},
+                {title: 'Name', details: enterprise.name},
+                {title: 'Registered Name', details: registered_name},
+                {title: 'Address', details: address},
+                {title: 'Location', details: location},
+                {title: 'Zip Code', details: zipcode},
+                {title: 'Email', details: email},
+                {title: 'Contact Person', details: contact_person},
+                {title: 'Contact Number', details: contact_number}
             ]
         else:
             renderable = {
-                'id': enterprise.id,
-                'name': enterprise.name,
-                'display_name': enterprise.display_name,
-                'registered_name': registered_name,
-                'address': address,
-                'location': location,
-                'zipcode': zipcode,
-                'email': email,
-                'contact_person': contact_person,
-                'contact_number': contact_number
+                'Enterprise Id': enterprise.id,
```

Comment:
Don't change the field name? Any specific reason?

Reply:
@jerynmathew suggested to change.

Reply:
so that It doesn't look like code variables